e2e.groovy

```diff
@@ -0,0 +1,51 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/ceph/ceph-csi'
```

Comment:
For testing you will need to change this repository URL and branch to your fork. Otherwise the old files (and no additional ones) will be checked out.

Reply:
Thanks, I'll make the required changes